Week1/homework/js2-exercises/3-hijackGoogleLogo.js

```diff
@@ -0,0 +1,7 @@
+function hijackGoogleLogo () {
+    let googleImage=document.querySelector('#hplogo')
```

Comment:
this selector didn't work on my google version it would be interesting to see if it works on yours. can you make a screenshot of the element with this id ?